.github/workflows/main.yml

```diff
@@ -47,12 +51,15 @@ jobs:
       run: .github/bin/dist-github-release.sh
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - if: github.event_name != 'pull_request'
+    - if: >-
+        github.event_name != 'pull_request' &&
+        github.repository == 'planetarium/libplanet'
       run: .github/bin/dist-nuget.sh
       env:
         NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
 
   docs:
+    if: (github.event_name == 'schedule' && github.repository == 'planetarium/libplanet') || (github.event_name != 'schedule')
```

Comment:
Since it's logically equivalent to the `build` job's `if` condition, and the hard-coded `'planetarium/libplanet'` is repeated, it would be better if the dup is eliminated using an [anchor].  I recommend you to read [this SO answer][1] if you aren't aware of YAML's this feature.

[anchor]: https://yaml.org/spec/1.2/spec.html#id2760569
[1]: https://stackoverflow.com/a/5954748/383405

Reply:
I tried to using anchor but I figured out that [GitHub Actions currently didn't support anchors](https://github.com/gurrpi/libplanet/runs/511133814). 
```
### ERRORED 13:39:08Z

- Your workflow file was invalid: The pipeline is not valid. .github/workflows/main.yml: Anchors are not currently supported. Remove the anchor 'condition'
```

Reply:
Okay, it can't be helped. 😇 Could you adjust the code style then, so that two equivalent `if` conditions look exactly same?